constants/index.ts

```diff
@@ -1,4 +1,5 @@
-export const APP_NAME = 'Elk'
+export const APP_NAME = 'Mozilla.social'
+export const APP_WEBSITE = 'https://mozilla.social'
```

Comment:
I would've felt bad defining a constant for something that gets used once if `APP_NAME` wasn't also only used once.